packages/dai-plugin-migrations/test/helpers/mocks.js

```diff
@@ -0,0 +1,54 @@
+import BigNumber from 'bignumber.js';
+import { RAY } from '../../src/constants';
+import { bytesToString } from './';
+
+export const globalSettlement = {
+  beforeCage: () => ({
+    live: async () => true,
+    tags: async () => new BigNumber(0),
+    fix: async () => new BigNumber(0)
+  }),
+  afterCage: () => ({
+    live: async () => false,
+    tags: async () => new BigNumber(0),
+    fix: async () => new BigNumber(0)
+  }),
+  afterCageCollateral: tags => ({
+    live: async () => false,
+    tags: async ilk => {
+      const ilkAsString = bytesToString(ilk);
+      return tags[ilkAsString]
+        ? RAY.times(new BigNumber(1).div(tags[ilkAsString]))
+        : new BigNumber(0);
+    },
+    fix: async () => new BigNumber(0)
+  }),
+  afterFlow: fixes => ({
+    live: async () => false,
+    fix: async ilk => {
+      const ilkAsString = bytesToString(ilk);
+      return fixes[ilkAsString]
+        ? RAY.times(fixes[ilkAsString])
+        : new BigNumber(0);
+    }
+  })
+};
```

Comment:
It'd be nice to not rely on mocks for testing global settlement. Planning to talk to Gonzalo about this. Deployment code suggests that leaving an authed user around to kick off settlement would be super not straight-forward, but maybe he can think of a better way?

Reply:
> hi Andrea, once the `ds-chief` is set as authority, you need to go through it in order to make changes to the system like Global Settlement
there is a way to import your own authority system and don't deploy a chief, but it shouldn't be hard to use the `ds-chief` directly

as it will not have any vote at all
with just 1 wei of MKR you could get one address elected as the new `hat`
```
seth send "$MCD_GOV" 'approve(address,uint256)' "$MCD_ADM" 1
seth send "$MCD_ADM" 'lock(uint256)' 1
seth send "$MCD_ADM" 'vote(address[] memory)' ["${ETH_FROM#0x}"]
seth send "$MCD_ADM" 'lift(address)' "$ETH_FROM"

```
that should leave your address as the `hat` of the chief
meaning that it has a `root` role where can execute a global settlement